### PR TITLE
Extract common <td> styles into components

### PR DESCRIPTION
**Pull request by zachmargolis**
Giving this a shot based on this comment: https://github.com/18F/identity-reporting/pull/89#discussion_r1126994693

### CODE REVIEW

File: src/components/table.tsx [hunk 1]
```diff
@@ -32,7 +64,7 @@ function Row({
   return (
     <tr>
       {row.map((d) => {
-        if (typeof d === "object" && d.type === "td") {
+        if (typeof d === "object" && (d.type === "td" || Object.values(Td).includes(d.type as any))) {
```
[the remark this answers is not in the document]
Comment (zachmargolis) [reply src/components/table.tsx:67]: The goal was that only `<td>` tags get passed through unwrapped. If we receive a `<span>` which is still a VNode, we want to wrap that still

[the remark this answers is not in the document]
Comment (aduth) [reply src/components/table.tsx:67]: I see, okay that makes sense.

**Comment by aduth** [comment]
I see a bunch of errors when previewing this:

```
Improper nesting of table. Your <td> should have a <tr> parent.<td data-csv="22169" className="table-number text-tabular text-right" style="undefined">..</td>
```

**Comment by zachmargolis** [comment]
> I see a bunch of errors when previewing this:

which page/report?

**Comment by aduth** [comment]
> > I see a bunch of errors when previewing this:
> 
> which page/report?

On "Account Deletions Report", when viewed locally, not in Federalist preview.

**Comment by zachmargolis** [comment]
> > > I see a bunch of errors when previewing this:
> > 
> > 
> > which page/report?
> 
> On "Account Deletions Report", when viewed locally, not in Federalist preview.

We chatted on Slack, we think this is a benign error where Preact's check for nesting is not thorough enough

**Pull request merged**